runtime/README.md

```diff
@@ -692,107 +708,504 @@ Command Code: `0x434B_4558` ("CKEX")
+| preamble_owner_lms_key     | u32[6]       | Owner LMS-SHA192-H15 key                                                    |
+| preamble_owner_ecc384_sig  | u32[24]      | Owner ECC384 signature                                                      |
+| preamble_owner_LMS_sig     | u32[1344]    | Owner LMOTS-SHA192-W4 signature                                             |
+| metadata_vendor_ecc384_sig | u32[24]      | Metadata Vendor ECC384 signature                                            |
+| metadata_vendor_LMS_sig    | u32[1344]    | Metadata Vendor LMOTS-SHA192-W4 signature                                   |
+| metadata_owner_ecc384_sig  | u32[24]      | Metadata Owner ECC384 signature                                             |
+| metadata_owner_LMS_sig     | u32[1344]    | Metadata Owner LMOTS-SHA192-W4 signature                                    |
+| metadata_header_revision    | u32          | Revision of the metadata header                                             |
+| metadata_header_reserved    | u32[3]       | Reserved                                                                    |
+| metadata_entry_entry_count | u32          | number of metadata entries                                                  |
+| metadata_entries             | MetaData[16] | The max number of metadata is 16 but less can be used                       |
 
 
 *Table: `AUTH_MANIFEST_FLAGS` input flags*
 
 | **Name**                  | **Value** |
-|---------------------------|-----------|
+| ------------------------- | --------- |
 | VENDOR_SIGNATURE_REQUIRED | 1 << 0    |
 
 *Table: `AUTH_MANIFEST_METADATA_ENTRY` digest entries*
 
 | **Name**      | **Type** | **Description**        |
-|---------------|----------|------------------------|
+| ------------- | -------- | ---------------------- |
 | digest        | u32[48]  | Digest of the metadata |
-| image\_source | u32      | Image source           |
+| image_source | u32      | Image source           |
 
 *Table: `SET_AUTH_MANIFEST` output arguments*
 
-| **Name**      | **Type** | **Description**
-| --------      | -------- | ---------------
-| chksum        | u32      | Checksum over other output arguments, computed by Caliptra. Little endian.
-| fips\_status  | u32      | Indicates if the command is FIPS approved or an error.
-
+| **Name**     | **Type** | **Description**                                                            |
+| ------------ | -------- | -------------------------------------------------------------------------- |
+| chksum       | u32      | Checksum over other output arguments, computed by Caliptra. Little endian. |
+| fips\_status | u32      | Indicates if the command is FIPS approved or an error.                     |
 
 ### AUTHORIZE_AND_STASH
 
+The SoC uses this command to request authorization of its various SoC images. This command has the option to receive the image hash directly from SoC or from an external source (e.g., SHA Acc).
```

Comment:
SHA Acc is deprecated in passive mode because the SHA is no longer expose on the AXI bus

Reply:
I think we still need to sort this out. This is a pretty annoying hw regression to deal in fw with since it impacts how the API surface works.